Source/Core/Core/PowerPC/Jit64Common/Jit64PowerPCState.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2010 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included./
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+#include "Core/PowerPC/PowerPC.h"
+
+// We offset by 0x80 because the range of one byte memory offsets is
+// -0x80..0x7f.
+#define PPCSTATE(x)                                                                                \
+  MDisp(RPPCSTATE, (int)((char*)&PowerPC::ppcState.x - (char*)&PowerPC::ppcState) - 0x80)
```

Comment:
I agree, not sure what @lioncash  will say but I personally think that would be better off for a separate commit, as the original code also had that issue.